src/App.tsx

```diff
@@ -2,24 +1,44 @@
+import React, { useState, useCallback } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const handlePageChange = useCallback((page: number) => {
+    setCurrentPage(page);
+  }, []);
```

Comment:
The `useCallback` hook is used here without any dependencies. This is redundant since the `setCurrentPage` function from `useState` does not change between renders. You can safely remove `useCallback` in this case.

Reply:
fixed